test/test_get_account_statement.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pytest
+
+import pytransferwise
+
+pytransferwise.api_key = os.getenv('TRANSFERWISE_API_KEY')
```

Comment:
Although `pytest test --forked` runs each test within a separate process, it looks like all top-level Python module code is evaluated before the `fork` takes place.

That means that this line is setting `pytransferwise.api_key` for all of the tests that run, which isn't what we want - we'd prefer it only to affect tests within this file (`test_get_account_statement.py`).

One suggestion for a way to fix this would be to configure and construct the `client` object via a fixture instead, like the `interval_start`, `interval_end` fixtures:

```python
@pytest.fixture
def client():
    pytransferwise.api_key = ...
    return ...
```

Reply:
@jayaddison, have changed this!